basil/HL/tti_ql355tp.py

```diff
@@ -5,7 +5,7 @@
 # ------------------------------------------------------------
 #
 
-from __future__ import print_function
```

Comment:
You remove Python 2 compatibility or why you do this?

Reply:
I would not remove for now. It does not cost much to keep it for us.